cg/server/api.py

```diff
@@ -216,10 +219,20 @@ def get_families():
     status: str = request.args.get("status")
     enquiry: str = request.args.get("enquiry")
     action: str = request.args.get("action")
+    pipeline: Pipeline = Pipeline(request.args.get("pipeline", Pipeline.MIP_DNA))
```

Comment:
Should this not be ```pipeline```

```suggestion
    pipeline: Pipeline = Pipeline(request.args.get("pipeline", pipeline))
```

Reply:
ah wait nvm the syntax is, get(<arg>, default) 

```suggestion
    pipeline: Pipeline = Pipeline(request.args.get("pipeline", default=Pipeline.MIP_DNA))
```

in that case maybe?